### ISSUE TITLE
unable to load dll 'questpdfskia' on Windows 11 .NET Framework C#

### ISSUE THREAD

**Issue by 2BDevTeam** [QuestPDF/QuestPDF#847]
**Describe the bug**
When I run the app, the following error appears: unable to load dll 'questpdfskia'.

**To Reproduce**
QuestPDF version is 2024.3.0

**Expected behavior**

**Screenshots**

**Environment**
What version of the library do you use? 
2024.3.0
What operating system do you use? (OS type, x64 vs x86 vs arm64)
ARM64

**Additional context**
I am using Windows 11 VM on Mac M1 with Parallels.

**Comment by Shahid-khan5** [comment]
I am facing the same issue

**Comment by MarcinZiabek** [comment]
Thank you for sharing your experience.

Due to the low adoption rate, QuestPDF does not support the Windows-arm64 platform. I suppose that you are running the Windows 11 virtual machine on Parallels. Therefore, you are likely using the arm64 version. Is that correct?

**Comment by Shahid-khan5** [comment]
No, I am not running it in virtual machine. Changing the app architecture to x64 fixed the issue. Thanks

**Comment by ahmad-syafrudin** [comment]
well, i forgot to set license in startup.cs

using .net 4.8 web mvc

**Comment by horros** [comment]
Ran into this now. I'm using Windows 11 with a Snapdragon Elite X processor.

**Comment by aathar** [comment]
<img width="301" alt="Image" src="https://github.com/user-attachments/assets/ca950790-0d20-4628-9ce5-dcdbad1f63df" />

Selecting the option x64 worked for me, and the project compiles and runs as expected.
Project properties -> Platform Target -> change "any CPU" to x64

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this issue due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

We are happy to share that Windows ARM64 support has been added in version 2026.6.0, which should resolve the issue you described.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
